src/libsyncengine/jobs/network/downloadjob.cpp

```diff
@@ -207,16 +207,15 @@ bool DownloadJob::handleResponse(std::istream &is) {
     } else {
```

Comment:
These changes are required for successful compilation on MacOSX but are not part, per se, of this PR. See earlier dedicated PR fixing the compilation issue.